### PR TITLE
Dependency updates for master

### PR DESCRIPTION
**Pull request by Lex-2008**
Update zlib from 1.2.12 to 1.2.13
Update openssl from 3.0.4 to 3.0.7
Update sasl2 from 2.1.27 to 2.1.28
Update openldap from 2.6.2 to 2.6.3
Update libcurl-hub from 7.83.1 to 7.86.0
Update apache from 2.4.53 to 2.4.54
Update git from 2.36.1 to 2.38.1
Update rsync from 3.2.4 to 3.2.7
Update postgresql from 14.3 to 15.1
Update php from 8.1.7 to 8.1.12
Update libcurl from 7.83.1 to 7.86.0

Updated libxml2 from 2.9.12 to 2.9.14 in PR #1143

Merge together:
https://github.com/cfengine/buildscripts/pull/1145
https://github.com/cfengine/nova/pull/1942
https://github.com/cfengine/core/pull/5104

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build exotics plz

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8840/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8840/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8840/

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8879/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8879/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8879/

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8880/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8880/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8880/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8881/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8881/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8881/

**Comment by vpodzime** [comment]
@cf-bottom jenkins again, this time with exotics, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8882/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8882/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8882/

**Comment by vpodzime** [comment]
@cf-bottom jenkins again, now that we have the SELinux fix for PG 15.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8884/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8884/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8884/

File: deps-packaging/openssl/cfbuild-openssl.spec [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+%define openssl_version 3.0.7
 
 Summary: CFEngine Build Automation -- openssl
 Name: cfbuild-openssl
 Version: %{version}
 Release: 1
 Source0: openssl-%{openssl_version}.tar.gz
-Patch0: 0005-aix-config-pm.patch
-Patch1: 0006-Add-latomic-on-AIX-7.patch
+# Patch0: 0005-aix-config-pm.patch
```
Comment (craigcomstock) [inline deps-packaging/openssl/cfbuild-openssl.spec:8]: remove comments for final version...

File: deps-packaging/zlib/cfbuild-zlib.spec [hunk 2]
```diff
@@ -14,21 +15,25 @@ AutoReqProv: no
...
   export MAKE=$MAKE_PATH
 fi
 
 ./configure --prefix=%{prefix}
 
 $MAKE
-%if %{?with_testsuite:1}%{!?with_testsuite:0}
-$MAKE check
-%endif
+
```
Comment (craigcomstock) [inline deps-packaging/zlib/cfbuild-zlib.spec:32]: remove comments in final version

**Review by vpodzime — COMMENTED** [review]
I guess we should also update the dependencies table here? Looks good to me otherwise.

**Comment by Lex-2008** [comment]
@cf-bottom can do it in a separate PR, after this one is merged.

This is done separately because we usually have several deps-updates PRs running in parallel, and them changing same lines in a table will result in merge conflicts.

**Comment by Lex-2008** [comment]
deps-table update PR: https://github.com/cfengine/buildscripts/pull/1156

**Pull request merged**